src/components/modules/readonly.ts

```diff
@@ -71,6 +71,13 @@ export default class ReadOnly extends Module {
 
     this.readOnlyEnabled = state;
 
+    /**
+     * Do not render UI if state is read-only and it's initial call
```

Comment:
what does it mean "do not render UI" ? please, update jsdoc to make it more straightforward